konnectivity-client/pkg/client/client.go

```diff
@@ -94,11 +94,11 @@ func (t *grpcTunnel) serve(c clientConn) {
 			return
 		}
 		if err != nil || pkt == nil {
-			klog.Warningf("stream read error: %v", err)
+			klog.V(5).Infof("stream read error: %v", err)
```

Comment:
Konnectivity-client is our client library, so its running in somewhere like the KAS. As such I think we should probably be using https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md for this package. As such this should be V2 or higher.

Reply:
I think that we should probably also go for https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent . I've changed some of the logging to use `InfoS`, but in this case as recommendation by this guide `klog.Warning{ln,f,Depth} -> klog.InfoS`

Reply:
What about enforcing the rules from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md to the entire repo and not only to this specific package? This would simplify things a lot.

Reply:
> What about enforcing the rules from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md to the entire repo and not only to this specific package? This would simplify things a lot.

+1 for the simplification.

Reply:
+1 for simplification. For server/agent I think a rule of thumb like abnormal behavior should be logged at a higher level than expected behavior. So handshakes/connections/packets are less important/interesting than shutdown requests which are less important/interesting than unexpectedly broken connections.